plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java

```diff
@@ -106,4 +118,14 @@ private static void validateDomainCompactionThreshold(int domainCompactionThresh
             }
         });
     }
+
+    private static void validateInsertBatchSize(int maxBatchSize, int maxAllowedBatchSize)
+    {
+        if (maxBatchSize < 1) {
+            throw new TrinoException(INVALID_SESSION_PROPERTY, format("Insert batch size (%s) must be greater than 0", maxBatchSize));
+        }
+        if (maxBatchSize > maxAllowedBatchSize) {
+            throw new TrinoException(INVALID_SESSION_PROPERTY, format("Insert batch size (%s) cannot exceed %s", maxBatchSize, maxAllowedBatchSize));
```

Comment:
Please use the actual session property name in the error message.

That way when someone is setting multiple session properties in a single statement it'll become more clear what is the error.

```suggestion
            throw new TrinoException(INVALID_SESSION_PROPERTY, format("%s must be greater than 0: %s", INSERT_BATCH_SIZE, maxBatchSize));
        }
        if (maxBatchSize > maxAllowedBatchSize) {
            throw new TrinoException(INVALID_SESSION_PROPERTY, format("%s cannot exceed %s: %s", INSERT_BATCH_SIZE, maxAllowedBatchSize, maxBatchSize));
```